### PR TITLE
Avtalegiro change email

### LINKED ISSUE #351: Add email for AvtaleGiro changes

### PR DESCRIPTION
**Pull request by philiphand**
Closes #351

### CODE REVIEW

**Comment by philiphand** [comment]
Kommer snart en commit!

**Comment by philiphand** [comment]
Klar for review nå!

File: src/custom_modules/mail.js [hunk 1]
```diff
@@ -563,6 +563,50 @@ async function sendTaxDeductions(taxDeductionRecord, year) {
   }
 }
 
+/** 
+ * @param {string} KID
+ * @param {"STOPPED" | "AMOUNT" | "CHARGEDAY" | "SHARES"} change What change was done
```
Comment (fellmirr) [inline src/custom_modules/mail.js:568]: Is this correct? I see that CANCELLED is used further down in the code f.ex.
I think the states are different for avtalegiro than for vipps

**Review by fellmirr — CHANGES_REQUESTED** [review]
One minor comment, looks good other than that :+1:

**Comment by philiphand** [comment]
The type was indeed wrong, all fixed now!

**Pull request merged**